sound/soc/sof/intel/hda.h

```diff
@@ -495,6 +495,15 @@ struct sof_intel_hda_dev {
 
 	int boot_iteration;
 
+	/*
+	 * DMA buffers for base firmware download. By default the buffers are
+	 * allocated once and kept through the lifetime of the driver.
+	 * See module parameter: keep_firmware_dma_buffer
```

Comment:
where is this module parameter defined?

Reply:
good catch, I renamed the parameter at the last time and missed this comment.